dashboard/app/models/levels_skill.rb

```diff
@@ -13,4 +13,13 @@
 class LevelsSkill < ApplicationRecord
   belongs_to :level
   belongs_to :skill
+
+  def seeding_key(seed_context)
```

Comment:
once you are serializing these to the .level file, you shouldn't need a seeding_key method -- the .level file can just contain a list of `skill.key` values.